implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -168,7 +168,8 @@ impl NodeManagerWorker {
                 .and_then(|p| {
                     if let Some(p) = p.cast::<Project>() {
                         node_manager
-                            .projects
+                            .projects()
+                            .unwrap()
```

Comment:
We should handle this gracefully, instead of panicing.

Reply:
Fixed https://github.com/build-trust/ockam/pull/4075/files#diff-5b3f34a98f0adf81c262908c1b6ee444f6e61e4242075b18860c9a57ea986e3fR168-R170